README.rst

```diff
@@ -0,0 +1,83 @@
+pyincore-incubator
+========
+
```

Comment:
All look good. Maybe we don't need this line? `pyincore-incubator ========`
![image](https://github.com/IN-CORE/pyincore-incubator/assets/63822845/21ccd969-e716-4189-adda-491a16157115)